### ISSUE TITLE
log size not filtered for insertion in database

### ISSUE THREAD

**Issue by vinpel** [php-censor/php-censor#239]
## Expected behavior

gracefully manage big log

## Actual behavior

```shell
PHP Fatal error:  Uncaught PDOException: SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'log' at row 1 in /app/php-censor/src/Store.php:164
Stack trace:
#0 /app/php-censor/src/Store.php(164): PDOStatement->execute()
#1 /app/php-censor/src/Store.php(128): PHPCensor\Store->saveByUpdate(Object(PHPCensor\Model\Build\GogsBuild), false)
#2 /app/php-censor/src/Logging/BuildDBLogHandler.php(62): PHPCensor\Store->save(Object(PHPCensor\Model\Build\GogsBuild))
#3 /app/php-censor/src/Logging/BuildDBLogHandler.php(53): PHPCensor\Logging\BuildDBLogHandler->flushData()
#4 [internal function]: PHPCensor\Logging\BuildDBLogHandler->__destruct()
#5 {main}
  thrown in /app/php-censor/src/Store.php on line 164

Fatal error: Uncaught PDOException: SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'log' at row 1 in /app/php-censor/src/Store.php:164
Stack trace:
#0 /app/php-censor/src/Store.php(164): PDOStatement->execute()
#1 /app/php-censor/src/Store.php(128): PHPCensor\Store->saveByUpdate(Object(PHPCensor\Model\Build\GogsBuild), false)
#2 /app/php-censor/src/Logging/BuildDBLogHandler.php(62): PHPCensor\Store->save(Object(PHPCensor\Model\Build\GogsBuild))
#3 /app/php-censor/src/Logging/BuildDBLogHandler.php(53): PHPCensor\Logging\BuildDBLogHandler->flushData()
#4 [internal function]: PHPCensor\Logging\BuildDBLogHandler->__destruct()
#5 {main}
  thrown in /app/php-censor/src/Store.php on line 164
```

## Steps to reproduce

lot of log

## Environment

* PHP Censor version master
* Operating System docker/php7-fpm
* PHP version 7.1
* MySQL/PostgreSQL version

**Comment by corpsee** [comment]
@vinpel Thanks. Are you use MySQL?

**Comment by corpsee** [comment]
@vinpel What are you meaning: 'gracefully manage big log'? We should cut log before DB insert?

**Comment by kromonos** [comment]
Had same problem here on MySQL 5.7. After altering column to LONGTEXT it works:
```sql
ALTER TABLE `build` CHANGE `log` `log` LONGTEXT CHARACTER SET utf8 COLLATE utf8_general_ci NULL DEFAULT NULL 
```

**Comment by corpsee** [comment]
@vinpel @kromonos I released version `1.0.2` with fix for MySQL column types. Also fix included in branches `release-1.0` and `master`.
